plugins/nf-google/src/main/nextflow/cloud/google/batch/GoogleBatchFusionAdapter.groovy

```diff
@@ -47,12 +47,7 @@ class GoogleBatchFusionAdapter implements GoogleBatchLauncherSpec {
 
     @Override
     List<Volume> getVolumes() {
-        return [
-                Volume.newBuilder()
-                        .setDeviceName('fusion')
-                        .setMountPath('/tmp')
-                        .build()
-        ]
```

Comment:
Not sure to understand this. Why is not mount added anymore?

Reply:
Because it is added here:

https://github.com/nextflow-io/nextflow/blob/e9679770b3580b619d58a2b15c3e410125518132/plugins/nf-google/src/main/nextflow/cloud/google/batch/GoogleBatchTaskHandler.groovy#L246-L251